backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlQuery.java

```diff
@@ -613,15 +623,107 @@ private Field<?> jsonAggregateSelect(
     return jsonField(table, column, tableAlias, select, filters, searchTerms, subAlias, fields);
   }
 
-  private Field<Object> jsonAggregateGroupBy(
+  //  private Field<Object> jsonGroupBySelect(
```

Comment:
remove? We have git for version control of code

Reply:
ja, oeps.